client/components/data/query-countries/README.md

```diff
@@ -0,0 +1,34 @@
+Query Countries
+====================
+
+`<QueryCountries />` is a React component used in managing network requests for country list for specific purposes.
+
+## Usage
+
+Render the component, passing `listType`. It does not accept any children, nor does it render any elements to the page. You can use it adjacent to other sibling components which make use of the fetched data made available through the global application state.
+
+```jsx
+import React from 'react';
+import { DOMAIN } from 'state/countries/constants'; 
```

Comment:
At first glance, it's not obvious what "DOMAIN" means in this context. I might have expected these constants to be within some grouping like `listTypes`.